portality/migrate/reapplication/delete_unreapplied.py

```diff
@@ -0,0 +1,41 @@
+from esprit import tasks, raw
+from portality import models
+
+'''Delete journals and corresponding articles with incomplete reapplications'''
+
+# Connection to the ES index
+conn = raw.make_connection(None, 'localhost', 9200, 'doaj')
```

Comment:
Ah you're using tasks.dump . No idea if there's a better way, but it seems reasonable, and presumably you've ran it successfully + we'll run it on staging eventually.